### ISSUE TITLE
ReplayGain support

### ISSUE THREAD

**Issue by CorePoint** [dkanada/gelli#97]
Hey,

would you consider adding [ReplayGain](https://en.wikipedia.org/wiki/ReplayGain) support?

Thank you for your work!

**Comment by masonh** [comment]
Possible duplicate of #18

**Comment by CorePoint** [comment]
You're right it's similar and I didn't see the issue before. I still would say that this feature request is more specific, because volume normalization could be reached trough different means.

Also if you are ever going to implement it, it would be awesome to have an automatic mode selection like this [foobar2k plugin](https://www.foobar2000.org/components/view/foo_dsp_replaygain) does it.

Wish you a good week!

**Comment by CorePoint** [comment]
Good news, Auxio which also uses exoplayer implemented replaygain.
Here is the issue in case you are interested in how it's solved in that app.
https://github.com/OxygenCobalt/Auxio/issues/7
